### ISSUE TITLE
modify inference_small function in resnet.py

### ISSUE THREAD

**Issue by striviz** [ry/tensorflow-resnet#13]
the original  code in resnet.py within inference_small function didn't have return values!!!so add the following code at its last

`logits=inference_small_config(x, c)
return logits`
